binaries/llm-cli/src/main.rs

```diff
@@ -256,6 +256,12 @@ fn interactive(
     let parameters = generate.inference_parameters(model.eot_token_id());
     let mut rng = generate.rng();
 
+    let stop_sequence = message_prompt_template
```

Comment:
I am assuming the message_prompt_template is something like:
```
User: {{PROMPT}}
```

But very open to suggestions here